app/src/main/java/com/github/se/travelpouch/ui/travel/EditTravelSettings.kt

```diff
@@ -304,9 +305,7 @@ fun EditTravelSettingsScreen(
                 label = { Text("End Date") },
                 placeholder = { Text("DD/MM/YYYY") },
                 modifier =
-                    Modifier.testTag("inputTravelEndTime")
-                        .fillMaxWidth()
-                        .padding(horizontal = 10.dp),
+                    Modifier.testTag("inputTravelEndTime").width(300.dp).padding(vertical = 4.dp),
```

Comment:
Here I don't understand why we would hard code the size? It's not always wrong to hard code some padding or sizes, 
I think it matters in this place, since we're wasting horizontal real estate in my opinion.
Might I suggest you replace with .fillMaxWidth(0.8f) for example if you want it not to take up the entire space while still being adaptable to multiple devices